yarn.lock

```diff
@@ -632,27 +622,6 @@ algolia-frontend-components@0.0.35:
     webpack "^2.2.1"
     webpack-dev-server "^2.4.1"
 
-"algoliasearch-v3@npm:algoliasearch@3.35.1":
```

Comment:
maybe this should be done in a separate PR, but it was wrongly changed in https://github.com/algolia/algoliasearch-helper-js/pull/898